README.md

```diff
@@ -2,17 +2,8 @@ This is not a full portage configuration. It contains only those parts that are
 necessary to set up a gentoo github mirror based sync system:
 
 * Configures Portage to sync via https://github.com/gentoo/gentoo.git URL.
-* Updates metadata-cache (the first update might take long, all subsequent ones not).
+* Updates metadata-cache.
 * Updates the dtd directory.
 * Updates the glsa directory.
 * Updates `herds.xml` file.
 * Updates the news directory.
-
-## Notes ##
```

Comment:
done in https://github.com/hasufell/portage-gentoo-git-config/commit/c0616bd6e3b290cc6a0c0a9efb219fa77c41ca75